modules/e2ee/E2EEContext.js

```diff
@@ -133,12 +133,29 @@ export default class E2EEcontext {
      * @param {Uint8Array | boolean} key - they key for the given participant.
      * @param {Number} keyIndex - the key index.
      */
-    setKey(participantId, key, keyIndex) {
+    setKeyBytes(participantId, key, keyIndex) {
         this._worker.postMessage({
-            operation: 'setKey',
+            operation: 'setKeyBytes',
             participantId,
             key,
             keyIndex
         });
     }
+
+    setKey(participantId, key, keyIndex, unique = false) {
+        this._worker.postMessage({
+            operation: 'setKey',
+            key,
+            keyIndex,
+            participantId,
+            unique
+        });
+    }
+
+    setKeyManagementMode(externallyManaged) {
```

Comment:
Maybe we can call this "initialize" instead, and pass an options object, so we have some extensibility for the future.